README.md

```diff
@@ -26,4 +26,4 @@ Pre-compiled jars can be found on the [Quilt maven](https://maven.quiltmc.org/re
 
 ### On the command line
 
-`java -cp enigma.jar cuchaz.enigma.command.Main`
+`java -cp enigma.jar command.org.quiltmc.enigma.Main`
```

Comment:
wrong package

Reply:
intellij had a field day trying to figure out how to rename things